indico/htdocs/js/indico/RegistrationForm/tpls/fields/text.tpl.html

```diff
@@ -2,10 +2,14 @@
     <input type="text"
         id="field-{{ section.id }}-{{ field.id }}"
         name="{{ fieldName }}"
-        size="{{ field.values.length || 60}}"
+        size="{{ field.length || 60}}"
```

Comment:
Don't we have now a better place to define default values for `RegistrationFormFieldData`?